pywbem/cim_operations.py

```diff
@@ -643,6 +661,24 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
             Note that not all situations can be handled within this timeout, so
             for some issues, operations may take longer before raising an
             exception.
+
+          use_pull_operations (:class:`py:bool`):
+            Controls the use of pull operations in any `Iter...()` methods.
+            The default value (`None`) will work on any WBEM server whether or
+            not it supports pull operations; first the pull operation is
+            attempted and if the WBEM server does not support pull operations,
+            the corresponding instance variable is set to `False` and the
+            traditional (non-pull) operation will be executed for the current
+            operation and all subsequent Iter...()operations for this
+            WBEM Connection.
+
+            `True` means that pull operations will be attempted; all errors
+            including `CIM_ERR_NOT_SUPPORTED` are returned to the caller. The
+            traditional operations will not be attempted.
+
+            `False` means that only the traditional operations will be used,
+            regardless of whether or not the WBEM server also supports
+            pull operations.
```

Comment:
The description is not explicit about the fact that the instance variable is set to True is the server supports pull operations, and we could do a few editorial improvements as well.

I suggest to replace the description with this one:

```
            Controls the use of pull operations in any `Iter...()` methods, by
            intializing a corresponding instance variable that is used during
            any `Iter...()` methods.
            The default value (`None`) will work on any WBEM server whether or
            not it supports pull operations.

            `None` means that the `Iter...()` methods will attempt a pull operation
            first, and if the WBEM server does not support it, will use a traditional
            operation. The corresponding instance variable will be adjusted to
            indicate what the WBEM server supports.

            `True` means that the `Iter...()` methods will only use pull operations.
            If the WBEM server does not support pull operations, a
            :exc:`~pywbem.CIMError` with status code `CIM_ERR_NOT_SUPPORTED`
            will be raised.

            `False` means that the `Iter...()` methods will only use traditional operations.
```